.jenkins/build_docker_images.Jenkinsfile

```diff
@@ -5,47 +5,46 @@ DCAPTOOLS_REPO = "https://dcapdockerciregistry.azurecr.io"
 DCAPTOOLS_REPO_CREDENTIAL_ID = "dcapdockerciregistry"
 
 def buildDockerImages() {
-    node("nonSGX") {
-        timeout(GLOBAL_TIMEOUT) {
-            stage("Checkout") {
-                cleanWs()
-                checkout scm
-            }
+	node("nonSGX") {
+		timeout(GLOBAL_TIMEOUT) {
+			stage("Checkout") {
+				cleanWs()
+				checkout scm
+			}
```

Comment:
nit: Indentation seems off

Reply:
Agreed. Better to use spaces to be consistent with your other file's indentation.